### PR TITLE
Add initial config profile writer sample

### PR DESCRIPTION
**Pull request by pecigonzalo**
*Issue #:* #112 

*Description of changes:* Add support to store multiple profiles in config

---

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws-actions/configure-aws-credentials/blob/master/CONTRIBUTING.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

---

I don't know what format this repo uses, VSCode by default formats like this.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -128,60 +153,97 @@ async function assumeRole(params) {
...
+  // Specifies the role to assume by this profile
+  config[profileName].role_arn = roleArn;
+
+  // region:
+  // Specifies the region for this profile
+  config[profileName].region = regionName;
+
+  fs.writeFileSync(awsConfigFile, ini.stringify(config));
+}
+
```
Comment (pecigonzalo) [inline index.js:230]: This is the main change

**Comment by jfagoagas** [comment]
Hi @pecigonzalo, I was just about to create a new PR to support this workflow, I understand you want to include the feature to save the assumed credentials in the `${HOME}/.aws/config`, right?

In my use case I need the following step because a tool within the workflow requires to load the config from the `${HOME}/.aws/config` so to configure a secure flow I have to do the following:

```yaml
     - name: Configure AWS Credentials
       uses: aws-actions/configure-aws-credentials@v1
       with:
          aws-region: ${{ env.AWS_REGION }}
          role-to-assume: ${{ env.AWS_ROLE }}
          role-session-name: session-role

     - name: Set AWS Credentials file
       run: |
          [[ ! -d "${HOME}/.aws" ]] && mkdir ${HOME}/.aws
          echo "[profile ${{ env.ENV }}]" >> ${HOME}/.aws/config && \
          echo "aws_access_key_id=${{ env.AWS_ACCESS_KEY_ID }}" >> ${HOME}/.aws/config && \
          echo "aws_secret_access_key=${{ env.AWS_SECRET_ACCESS_KEY }}" >> ${HOME}/.aws/config && \
          echo "aws_session_token=${{ env.AWS_SESSION_TOKEN }}" >> ${HOME}/.aws/config
```

Thanks!

**Comment by bconnorwhite** [comment]
@jfagoagas Being able to set a custom path for the .aws directory would also be great. As an example with this action you need .aws to be in the action workspace, since ~/.aws isn't mounted to the image:
https://github.com/dflook/terraform-github-actions/issues/13#issuecomment-673612312

**Comment by jfagoagas** [comment]
@bconnorwhite totally agree, it would be great!

**Comment by pecigonzalo** [comment]
Agree with all comments and features, I opened this to confirm the approach with the contributors in the linked issue. If they are happy with it, I'll gladly support the use cases you highlighted

File: index.js (hunk 1 above)
Comment (pecigonzalo) [reply index.js:230]: The config needs to point to a webIdentityToken file, which is already supported. We just need to point to it from multiple profiles.

**Comment by pecigonzalo** [comment]
Closing in favor of https://github.com/aws-actions/configure-aws-credentials/pull/633

**Pull request closed without merging**